ZotBinsAPI.py

```diff
@@ -282,7 +289,94 @@ def get_obervation_stats():
         print(e)
         return str(e)
 
+@app.route('/mocked/observation/get', methods=['GET'])
+def mocked_get_observation():
+    """ 
+    An API endpoint that mocks observation GET requests.
+
+    Used for web and mobile applications to simulate
+    an identical GET request for real data.
+
+    Must pass all required params.
+
+    Possible sensor types = ["B", "D", "W", "DI"]
+        "B"  - Breakbeam
+        "D"  - Distance (ultrasonic sensor)
+        "W"  - Weight (load cell)
+        "Di" - Diversion Rate
+
+    @param   sensor_id, format = "ZBin" + [SENSOR_ID] + [SENSOR_TYPE]
+    @param   start_timestamp, format = "YYYY-MM-DD HH:MM:SS"
+    @param   end_timestamp, format example = "2020-02-01 9:30:01"
+    @return  JSON-ified array of mocked datapoints
+    """
+    try:
+        con = pymysql.connect(config.host, config.user, config.pw, config.db, cursorclass=pymysql.cursors.DictCursor)
+        ret = []
+        if request.method == 'GET':
```

Comment:
unnecessary. Already specified this is a GET endpoint.

Reply:
Technically true, the API won't run because it only accepts `GET` methods.

I intended on the code to be more robust so that's why I have this explicit `if request.method == 'GET'`, however it might actually become a source of confusion (e.g., "So this API can also take methods other than `GET`...?")

Will remove this.